packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1782,7 +1796,7 @@ export default class NewVm extends BaseComponent {
               <Item label={_('secureBoot')}>
                 <Toggle onChange={this._toggleState('secureBoot')} value={secureBoot} />
               </Item>
-              {secureBoot && pool !== undefined && !pool.secureBootSetup && (
```

Comment:
Is the previous API still valid? Don't we need to check the host's version in `xapi-object-to-xo.js`?

Reply:
There was no previous API. There was a temporary hack until the API was available, which was not supposed to go into production but eventually did. So here we're switching to the right API.

Reply:
Ok so in that case, should we remove this entirely? https://github.com/vatesfr/xen-orchestra/blob/c9699c8f02758181fdb0a9c2c5f2714c8b615825/packages/xo-server/src/xapi-object-to-xo.mjs#L102

Reply:
yes